app/models/spree_avatax/shared.rb

```diff
@@ -28,6 +28,24 @@ def taxable_order?(order)
       order.line_items.present? && order.ship_address.present?
     end
 
+    def get_tax(params)
+      with_timeout do
+        tax_svc.gettax(params)
+      end
+    end
+
+    def post_tax(params)
+      with_timeout do
+        tax_svc.posttax(params)
+      end
+    end
+
+    def cancel_tax(params)
+      with_timeout do
+        tax_svc.canceltax(params)
+      end
+    end
+
     def tax_svc
```

Comment:
Yah it could but I was thinking it's probably nice to be able to access this from a console & etc, or from custom code if someone using Avatax is using an endpoint that we're not covering.